### DISCUSSION TITLE
Why delete doesn't just delete the articles instead of just using a strikethrough?

### DISCUSSION THREAD

**Discussion by Tony20221** [martinrotter/rssguard#1436] [Q&A]
When I select articled and choose delete, why don't they actually get deleted and gone from the view? Instead of just striking through the entries.

**Comment by Ac314** [comment]
I guess they are moved to Recycled Bin? So they are not completely eradicated yet but still available in Recycled Bin.

**Reply by Tony20221** [reply]
I want them removed from the UI completely. They just make the UI very busy and it's very hard to discern which ones are deleted and which are not.
They need to be gone from the screen. If I want to restore them then I can go to where I can restore them.

**Reply by martinrotter** [reply]
Hello Tony.

Yes, the "deleted" items are strikedthrough and are not removed from the list, until you select another feed/item. There are some technical reasons for it, partly due to performance partly as relicts from old code.

Not 100% sure, but the feature could be hypothetically added.

So, I any relevant and good quality pull requests implementing this feature are welcomed, either from @Tony20221 or anybody else interested in this.
